bot/backend/apis/music.py

```diff
@@ -0,0 +1,50 @@
+"""interacting with the music api"""
+
+from decouple import config
+from loguru import logger
+import tekore
```

Comment:
not to be a wet blanket but `import x`s are supposed to come before the `from y import z` style imports as I'd shown in my suggested changes 😅